lldb/include/lldb/Utility/StructuredData.h

```diff
@@ -366,10 +396,10 @@ class StructuredData {
   class String : public Object {
   public:
     String() : Object(lldb::eStructuredDataTypeString) {}
-    explicit String(llvm::StringRef S)
-        : Object(lldb::eStructuredDataTypeString), m_value(S) {}
+    explicit String(llvm::StringRef s)
+        : Object(lldb::eStructuredDataTypeString), m_value(s) {}
 
-    void SetValue(llvm::StringRef S) { m_value = std::string(S); }
+    void SetValue(llvm::StringRef s) { m_value = std::string(s); }
```

Comment:
The changes to adjust the name of the variable are unnecessary and make it harder to navigate commit history. Please revert it.

Reply:
This improves readability/consistency within the file. Agreed about navigation of commit history. Will revert.